misc/config_tools/schema/checks/pre_launched_vm_support.xsd

```diff
@@ -52,4 +52,20 @@ https://github.com/projectacrn/acrn-hypervisor/issues if you meet this.</xs:docu
     </xs:annotation>
   </xs:assert>
 
+  <xs:assert test="(count(vm[vm_type = 'PRE_RT_VM']) = 1 and count(vm[vm_type='POST_RT_VM']) = 0)
+                   or (count(vm[vm_type='PRE_RT_VM']) = 0 and count(vm[vm_type='POST_RT_VM']) &gt;= 0)">
+    <xs:annotation acrn:severity="error">
+      <xs:documentation>There should not be both pre-launched RTVM and post-launched RTVM.
+And two or more pre-launched RTVM are not allowed.
+
+This assertion checks below invalid scenario:
+
+- Two or more pre-launched RTVM
+- One pre-launched RTVM and one or more post-launched RTVM
+
+To fix this error, double check the scenario xml file to ensure it is valid.</xs:documentation>
```

Comment:
```suggestion
To fix this error, double check the scenario XML file to ensure it is valid.</xs:documentation>
```

Reply:
If this is a message printed out by the UI, it may be best to ask the user to modify these values in the tool itself, instead of referring to the scenario XML which the user shouldn't touch directly.

Reply:
Yes, the message can be printed out by the UI if the error is met. The message will be updated.